classytags/helpers.py

```diff
@@ -83,7 +84,10 @@ def render_tag(self, context, **kwargs):
             safe_context.update(**data)
             output = render_to_string(template, safe_context)
         else:
-            data = context.new(self.get_context(context, **kwargs))
+            new_context = context.new(
+                flatten_context(self.get_context(context, **kwargs))
+            )
+            data = flatten_context(new_context)
```

Comment:
It's the context passed to https://github.com/ojii/django-classy-tags/blob/master/classytags/core.py#L138